### PR TITLE
IGAPP-712: Add debug documentation for web

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -17,13 +17,26 @@ React JS WebApp for [Integreat](https://integreat-app.de), [Malte](https://www.m
...
 
 ### Run the App
 
 - Run `yarn start` to start a local webpack dev server.
 - Start coding :)
 
+#### Debug the App (in IntelliJ)
+
+- Create a JavaScript Debug run configuration.
+  - Set the Url to `http://localhost:9000/`
```
Comment (klinzo) [inline web/README.md:31]:
```suggestion
  - Set the url to `http://localhost:9000/`
```

File: web/README.md [hunk 2]
```diff
@@ -17,13 +17,26 @@ React JS WebApp for [Integreat](https://integreat-app.de), [Malte](https://www.m
...
+  Configurations.
 
 ### Run the App
 
 - Run `yarn start` to start a local webpack dev server.
 - Start coding :)
 
+#### Debug the App (in IntelliJ)
+
+- Create a JavaScript Debug run configuration.
```
Comment (klinzo) [inline web/README.md:30]: Can't we create that in the project?

File: web/README.md [hunk 3]
```diff
@@ -17,13 +17,26 @@ React JS WebApp for [Integreat](https://integreat-app.de), [Malte](https://www.m
...
+  - Set the Url to `http://localhost:9000/`
+  - Select Chrome as Browser (debugging only works with chrome)
+- Run `yarn start` to start a local webpack dev server.
+- Start the Debug configuration in Debug mode
+- You can now Debug the webapp directly in IntelliJ
+
+
```
Comment (klinzo) [inline web/README.md:37]:
```suggestion
```

File: web/README.md [hunk 4]
```diff
@@ -17,13 +17,26 @@ React JS WebApp for [Integreat](https://integreat-app.de), [Malte](https://www.m
...
 - Start coding :)
 
+#### Debug the App (in IntelliJ)
+
+- Create a JavaScript Debug run configuration.
+  - Set the Url to `http://localhost:9000/`
+  - Select Chrome as Browser (debugging only works with chrome)
+- Run `yarn start` to start a local webpack dev server.
+- Start the Debug configuration in Debug mode
+- You can now Debug the webapp directly in IntelliJ
```
Comment (klinzo) [inline web/README.md:35]: Best use consistent punctuation :)

**Review by SteffiStuffel — APPROVED** [review]

File: web/README.md (hunk 2 above)
Comment (Taggotty) [reply web/README.md]: Already exists apparently

File: web/README.md [hunk 5]
```diff
@@ -24,6 +24,14 @@ _Make sure you read and followed the steps in the [general README](../README.md#
 - Run `yarn start` to start a local webpack dev server.
 - Start coding :)
 
+#### Debug the App (in IntelliJ)
+
+_Make sure that `sourceMap` is set to true in the `tsconfig.json`_
+
+- Run `yarn start` to start a local webpack dev server.
+- Start the JavaScript Debug with Chrome configuration in debug mode.
+- You can now Debug the webapp directly in IntelliJ.
```
Comment (klinzo) [inline web/README.md:33]:
```suggestion
- You can now debug the webapp directly in IntelliJ.
```

File: web/README.md [hunk 6]
```diff
@@ -24,6 +24,14 @@ _Make sure you read and followed the steps in the [general README](../README.md#
 - Run `yarn start` to start a local webpack dev server.
 - Start coding :)
 
+#### Debug the App (in IntelliJ)
+
+_Make sure that `sourceMap` is set to true in the `tsconfig.json`_
+
+- Run `yarn start` to start a local webpack dev server.
+- Start the JavaScript Debug with Chrome configuration in debug mode.
```
Comment (klinzo) [inline web/README.md:32]: Is this a run configuration? Could you wrap the name of the config in ``?

**Review by klinzo — APPROVED** [review]

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**